bls/src/main/java/org/dashj/bls/DASHJBLSJNI.java

```diff
@@ -0,0 +1,293 @@
+/* ----------------------------------------------------------------------------
+ * This file was automatically generated by SWIG (http://www.swig.org).
+ * Version 4.0.2
+ *
+ * Do not make changes to this file unless you know what you are doing--modify
+ * the SWIG interface file instead.
+ * ----------------------------------------------------------------------------- */
+
+package org.dashj.bls;
+
+public class DASHJBLSJNI {
+  public final static native long new_Uint8Vector__SWIG_0();
+  public final static native long new_Uint8Vector__SWIG_1(long jarg1, Uint8Vector jarg1_);
+  public final static native int Uint8Vector_capacity(long jarg1, Uint8Vector jarg1_);
+  public final static native void Uint8Vector_reserve(long jarg1, Uint8Vector jarg1_, int jarg2);
+  public final static native boolean Uint8Vector_isEmpty(long jarg1, Uint8Vector jarg1_);
```

Comment:
This class has a list of native wrappers that will execute all of the BLS functions in all classes and some helper classes that are generated by the SWIG tool.  The first parameter is generally a `long` which is the `this` pointer of the C++ object.